analyticsdataserver/settings/test.py

```diff
@@ -33,7 +33,7 @@
 }
 
 ANALYTICS_DATABASE = 'analytics'
-ANALYTICS_DATABASE_V1 = 'analytics_v1'
+ANALYTICS_DATABASE_V1 = None
```

Comment:
this might fail the router tests, not sure

Reply:
I checked and it does not